Common/src/Common/Form/Elements/InputFilters/Phone.php

```diff
@@ -20,7 +20,7 @@ class Phone extends LaminasElement implements InputProviderInterface
      */
     public function init(): void
     {
-        $this->setAttribute('pattern', '\d(\+|\-|\(|\))*');
+        $this->setAttribute('pattern', '^$|^[0-9][0-9 ]*$');
```

Comment:
Are we intentionally changing the allowed characters that this pattern intended to enforce? 

The new pattern removes `-` `+` `(` `)` characters which probably exist in the data. Without a database patch to update those characters out or at least knowing how many this will affect, may cause quite a bit of pain for users when updating existing entries via. the frontend.

I'd vote to fix it and keep it like-for-like.

Reply:
It's still not right. HTML pattern doesn't need start/end markers.

Try `\d(\+|-|\(|\))*`. The issue is the `-` is escaped when it doesn't need to be I'd imagine.